### ISSUE TITLE
test_target_teams_distribute_collapse.F90 – wrong map flag

### ISSUE THREAD

**Issue by tob2** [SOLLVE/sollve_vv#138]
In `test_target_teams_distribute_collapse.F90`'s `test_collapse2`, the variable  `b`'s initial value is used in the loop (zero initialized) but it is only marked as `from:` instead of `tofrom:`.

**Comment by tob2** [comment]
Namely:
```
--- a/tests/4.5/target_teams_distribute/test_target_teams_distribute_collapse.F90
+++ b/tests/4.5/target_teams_distribute/test_target_teams_distribute_collapse.F90
@@ -93,3 +93,3 @@
             !$omp target teams distribute map(to: a(1:N, 1:N, 1:N)) &
-            !$omp& map(from: b(1:N, 1:N, 1:N+1)) collapse(2)
+            !$omp& map(tofrom: b(1:N, 1:N, 1:N+1)) collapse(2)
             DO x = 1, N
```

**Comment by tob2** [comment]
Pull request: #139

**Comment by tob2** [comment]
Now fixed (→ #139 )
